cpu/nrf51822/include/nrfmin.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        drivers_nrf51822_nrfmin NRF Minimal Radio Driver
+ * @ingroup         drivers
+ * @brief           Minimal driver for the NRF51822 radio
+ *
+ * This driver enables the use of the NRF51822 radio in a IEEE802.15.4 like
+ * fashion. In the current state, the driver is only be meant to be used with
+ * the netdev/netapi based network stack, while only being able to communicate
+ * with other NRF51822 devices using the same driver.
+ *
+ * The driver is using a Nordic proprietary physical layer, configured to for a
+ * bitrate of 2Mbit. The payload length is set to a maximum length of 250 byte.

```

Comment:
Good point, but I say let's not get too academic here... I tested with cheap China modules and 250 byte packets, and that worked fine... So in my opinion its fine to go a little out of spec here...